cylc/flow/scripts/cylc.py

```diff
@@ -34,7 +34,7 @@ def pythonpath_manip():
         for item in os.environ['CYLC_PYTHONPATH'].split(os.pathsep):
             abspath = os.path.abspath(item)
             sys.path.insert(0, abspath)
-    elif 'PYTHONPATH' in os.environ:
+    if 'PYTHONPATH' in os.environ:
         os.environ['CYLC_PYTHONPATH'] = os.environ['PYTHONPATH']
```

Comment:
This now clobbers `CYLC_PYTHONPATH` if it was already set

Reply:
```suggestion
    elif 'PYTHONPATH' in os.environ:
```